LICENSE.md

```diff
@@ -0,0 +1,17 @@
+The MIT License (MIT)
+=====================
+
+Copyright © 2019 eLife Sciences Publications, Ltd
```

Comment:
could write 2020 because there is no content from 2019 here, but I Am Not A Lawyer and I'd keep it identical to everything else

Reply:
🤷‍♂ Think 2020 is fine.